### ISSUE TITLE
No Bid Responses in 1.0 Branch

### ISSUE THREAD

**Issue by GLStephen** [prebid/Prebid.js#1890]
I'm seeing no responses return from getBidResponses in our 1.0 branch where we are testing new analytics functionality. There are response events, and responses are definitely happening. So, I did some digging and found that the following in prebid.js was returning nothing:

```
$$PREBID_GLOBAL$$.getBidResponses
```

So, I ran:

```
gulp test --file "test/spec/auctionmanager_spec.js"
```

The result was this:

```
HeadlessChrome 0.0.0 (Linux 0.0.0) auctionmanager.js getKeyValueTargetingPairs No bidder level configuration defined - default FAILED
        AssertionError: {} deepEqual { hb_bidder: 'appnexus',
          hb_adid: '1adId',
          hb_pb: 5.5,
          hb_size: '300x250' }
            at context.<anonymous> (webpack:///test/spec/auctionmanager_spec.js:63:13 <- test/spec/auctionmanager_spec.js:3616:14)
HeadlessChrome 0.0.0 (Linux 0.0.0): Executed 19 of 19 (1 FAILED) (0.056 secs / 0.028 secs)
```

I was getting Responses last week before Thanksgiving, so my gut tells me something changed in one of these two commits: 

https://github.com/prebid/Prebid.js/commit/1fecaf9515fe553d0cab1c4c48506e01381b105e#diff-9a9e476b9246c1fdeb97faaf94fcbde2

https://github.com/prebid/Prebid.js/commit/6b387f347de11ab503a1cd16adf3f6ea343a023e#diff-9a9e476b9246c1fdeb97faaf94fcbde2

I'm not 100% familiar with all of the testing available in the project, so I'm not sure if that error is critical or a red herring. A simple "gulp test" is much more positive about the state of things. Saying 776 of 782 with 6 skipped.

**Comment by jaiminpanchal27** [comment]
@GLStephen Yes the bug got introduced with that winningbid committ. I have pushed a fix here #1689

**Comment by jaiminpanchal27** [comment]
Fixed in #1689
